roles/include_components/tasks/track_podman_component.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Assert repo_url and commit_id
+  assert:
+    that:
+      - item.0 is defined
+      - item.1 is defined
+      - item.0 | regex_match('^git@.*\.git$')   # Assert that item.0 is a valid repo_url
+      - item.1 | regex_match('^[0-9a-f]{40}$')  # Assert that item.1 is a valid commit_id
```

Comment:
With this format is it possible to add multiple components or only one at a time?

Reply:
sure, please check main.yml